.github/workflows/build-plugin-archive.yml

```
@@ -168,32 +198,77 @@ jobs:
           registry-url: ${{ inputs.NPM_REGISTRY_DOMAIN }}
           cache: ${{ env.NODE_CACHE_MODE }}
 
+      - name: Install Composer Asset Compiler
+        if: steps.composer-tools.outputs.assets-compiler == '0' && steps.check-for-package.outputs.has_package == 'true'
+        run: |
+          composer global config --no-plugins --no-interaction allow-plugins.inpsyde/composer-assets-compiler true
+          composer global require inpsyde/composer-assets-compiler
       - name: Run Composer Asset Compiler
         if: steps.composer-tools.outputs.assets-compiler == '0' && steps.check-for-package.outputs.has_package == 'true'
         run: composer compile-assets ${{ inputs.COMPILE_ASSETS_ARGS }}
```

Can we merge this into one step?
```suggestion
      - name: Install and run Composer Asset Compiler
        if: steps.composer-tools.outputs.assets-compiler == '0'
        run: |
          composer global config --no-plugins --no-interaction allow-plugins.inpsyde/composer-assets-compiler true
          composer global require inpsyde/composer-assets-compiler
          composer compile-assets ${{ inputs.COMPILE_ASSETS_ARGS }}
```

Note: I already removed the `has_package` check (see comment above).

---

Okay makes sense if you prefer it that way